src/Documentation/Getting-Started-With-Orleans/Grains.md

```diff
@@ -4,40 +4,71 @@ title: Grains
 ---
 
 
-## Grains (Actors): Units of Distribution
+### Grains
 
-Distributed applications are inherently concurrent, which leads to complexity. One of the things that makes the actor model special and productive is that it helps reduce some of the complexities of having to grapple with concurrency.
+Grains are the key primitives of the Orleans programming model.
+Grains are the building blocks of an Orleans application, they are atomic units of isolation, distribution, and persistence.
+Grains are objects that represent application entities.
+Just like in the classic Object Oriented Programming, a grain encapsulates state of an entity and encodes its behavior in the code logic.
+Grains can hold references to each other and interact by invoking each other’s methods exposed via interfaces. 
 
-Actors accomplish this in two ways:
+Orleans greatly simplify building a scalable application and eliminate most of the concurrency challenges 
```

Comment:
simplify -> simplifies, eliminate -> eliminates.

Alternatively, Orleans -> Orleans aims to